src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/overview/overview.component.ts

```diff
@@ -64,6 +80,18 @@ export class OverviewComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit() {
+    this.orchService.status().subscribe((status: OrchestratorStatus) => {
+      this.orchAvailable = status.available;
+    });
+
+    this.rbdMirroringService.subscribeSummary((data) => {
+      this.rbdPoolExists = data.content_data.pools.length > 0;
+    });
+
+    this.cephServiceService.list('rbd-mirror').subscribe((data) => {
+      this.rbdServiceExists = data.length > 0;
+    });
```

Comment:
I went the other way because if I have to go like this then I'll have to nest the *ngIf in two ng-containers. But I made the changes with async pipe now.